scripts/ci.ps1

```diff
@@ -0,0 +1,69 @@
+param(
+    # CMake Version to test with
+    [Parameter()]
+    [string]
+    $CMakeVersion = "3.10.0"
+)
+$ErrorActionPreference = "Stop"
+
+# The root directory of our repository:
+$REPO_DIR = Split-Path (Split-Path $MyInvocation.MyCommand.Source -Parent) -Parent
```

Comment:
I use `$MyInvocation` because it has the fully qualified Path. Should be able to derive the full path from `$PSScriptRoot`, I was just too lazy to write that code.

I'm also unsure if I can get away with PS3. I'm writing these against PS6, and haven't updated the AppVeyor builds yet. I'm always surprised by how much further ahead PS6 is from what is distributed on Windows.

Reply:
As does `$PSScriptRoot`. It just wasn't available in older versions.